lib/biokbase/catalog/version.py

```diff
@@ -1,2 +1,2 @@
 # File that simply defines version information
-CATALOG_VERSION = '2.1.3'
\ No newline at end of file
+CATALOG_VERSION = '2.2.0'
```

Comment:
The version here seems ok.  Maybe the version in Impl.py is irrelevant?

Reply:
Yup